src/desktop/lib/preload/Electron.js

```diff
@@ -44,6 +44,9 @@ let locales = {
 let onboardingSeed = null;
 let onboardingGenerated = false;
 
+// Use a different keychain entry for development versions
+const KEYTAR_SERVICE = currentWindow.isPackaged ? 'Trinity wallet' : 'Trinity wallet (dev)';
```

Comment:
`isPackaged` is available only starting from Electron `3.0`. To update to that we need to update the binary package builds to support node `10.2.0`. I have this in my next weeks todo.

This  merge should then wait till then, or we could change the check to `process.env.NODE_ENV === "development" ? 'Trinity wallet (dev)' : 'Trinity wallet'`

Reply:
Ah, I didn't realize that. I've changed to `process.env` in df850f3